certified-connectors/SignatureAPI/apiDefinition.swagger.json

```diff
@@ -0,0 +1,617 @@
+          {
+            "name": "envelopeId",
+            "x-ms-summary": "Envelope ID",
+            "description": "The ID of the envelope.",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Envelope in progress"
+          },
+          "200": {
+            "description": "Envelope is completed",
+            "schema": {
+              "$ref": "#/definitions/Envelope"
+            }
+          },
+          "400": {
+            "description": "Envelope is not completed"
+          }
+        }
+      }
+    },
+    "/deliverables/{deliverableId}": {
+      "get": {
+        "operationId": "GetDeliverable",
+        "summary": "Get a deliverable",
+        "description": "Retrieves a deliverable using its ID. Use it to download a signed copy of the envelope.",
+        "x-ms-visibility": "important",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "deliverableId",
+            "x-ms-summary": "Deliverable ID",
+            "description": "The ID of the deliverable.",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Deliverable retrieved",
+            "schema": {
+              "$ref": "#/definitions/Deliverable"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "EnvelopeId": {
+      "type": "string",
+      "x-ms-summary": "Envelope ID",
+      "description": "The ID of the envelope."
+    },
+    "EnvelopeTitle": {
+      "type": "string",
+      "x-ms-summary": "Envelope Title",
+      "description": "The title of this envelope, which will be shown to the recipients."
+    },
+    "EnvelopeMessage": {
+      "type": "string",
+      "x-ms-summary": "Envelope Message",
+      "description": "A message that will be sent to the recipients."
+    },
+    "EnvelopeSender": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "x-ms-summary": "Sender Name",
+          "description": "The name of the sender.",
+          "x-ms-visibility": "important"
+        },
+        "email": {
+          "type": "string",
+          "x-ms-summary": "Sender Email",
+          "description": "The email address of the sender.",
+          "x-ms-visibility": "important"
+        }
+      }
+    },
+    "EnvelopeStatus": {
+      "type": "string",
+      "x-ms-summary": "Envelope Status",
+      "description": "The status of the envelope.",
+      "x-ms-visibility": "important",
+      "enum": [
+        "draft",
+        "processing",
+        "in_progress",
+        "completed",
+        "failed",
+        "canceled"
```

Comment:
Could you please fix the typo?

Reply:
Thanks for the observation. We are using _"canceled"_ instead of _"cancelled"_ because we are adhering to American English spelling conventions for our strings.

> While both canceled and cancelled are acceptable for the past tense of cancel, the version with one L is more common in American English, while the version with two L's is more common in British English. American English typically only doubles the consonant when the stress is on the syllable attached to the suffix, as in remit and remitting.
[Source](https://www.merriam-webster.com/grammar/canceled-or-cancelled)